src/main/java/de/rytrox/varo/moderation/SpectateCommand.java

```diff
@@ -0,0 +1,53 @@
+package de.rytrox.varo.moderation;
+
+import de.rytrox.varo.utils.CommandHelper;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.TabExecutor;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SpectateCommand implements TabExecutor {
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if(sender instanceof Player && sender.hasPermission("varo.admin.moderator")) {
+            if(args.length > 0) {
+                Player target = Bukkit.getPlayer(args[0]);
+
+                if(target != null) {
+                    if(!target.hasPermission("varo.admin.moderator")) {
+                        ((Player) sender).teleport(target);
+
+                        sender.sendMessage(ChatColor.translateAlternateColorCodes('&', "&8[&6Varo&8] &7Du beobachtest nun den &2Spieler &a" + target.getDisplayName()));
```

Comment:
Bei der Nachricht würde ich davon ausgehen, dass der Moderator jetzt in der Spieler-Sicht ist. Aber das ist ja nicht der Fall oder? Daher würde ich sagen, entweder den Moderator dann in die Spielersicht setzten oder die Nachricht ändern zu (du wurdest zu so und so teleportiert)